layers/+distribution/spacemacs-base/funcs.el

```diff
@@ -336,7 +336,7 @@ argument takes the kindows rotate backwards."
 
 ;; from http://dfan.org/blog/2009/02/19/emacs-dedicated-windows/
 (defun spacemacs/toggle-current-window-dedication ()
-  "Toggle dedication state of a window."
+  "Toggle dedication state of a window (dedicated windows cannot be reused by a mode)."
```

Comment:
"cannot be reused by a mode" is not accurate, directly referring to `set-window-dedicated-p` or `window-dedicated-p` will be better, because their docstrings already contain correct and accurate info. Something like:
```elisp
"Toggle dedication state of a window.
See `set-window-dedicated-p' for details."
```